### DISCUSSION TITLE
Ambient Light Control

### DISCUSSION THREAD

**Discussion by pqhf5kd** [MonitorControl/MonitorControl#775] [Q&A] [answered]
I have an external monitor connected to an M1 MBA and have started using Monitor.Control this morning.

I have turned on 'Sync brightness changes...' and it works as expected when using Control Centre to adjust the internal display and when the ambient light changes.

1. The screen brightness keys on the MBA KB only control the monitor on which the mouse is positioned and the changes are not synced. This is expected?

If this is expected, it would be a useful feature to blank the internal screen while watching content on the external monitor.

2. _But_ it appears that ambient light control doesn't work when the internal screen is off/at 0% brightness. I suppose that this is macOS related and expected, because the user overrides the brightness and sets it to 0%, automatic control should be disabled.

3. Is it possible to 'zero brightness' the internal display AND have ambient control of the external?

**Comment by pqhf5kd** [comment]
On second thought, maybe that isn't a problem. 

Dim the ambient light first, macOS will react and then MonitorControl will sync the external display to the correct level.
Use the KB controls to blank the internal display.
Start the video.

**Comment by waydabber** [comment, marked as answer]
1. Yes, this is as intended. You can change this in Preferences however so the keyboard controls all displays.
2. Yes, when the internal screen is at 0%, the ambient light sensor does not change the screen.
3. Not at the moment. MC does not actually read the ambient light sensor (it could be done on M1 machines but this is not implemented), but the brightness changes of the internal screen which may or may not react to the ambient light sensor depending on settings and current brightness.
